content/docs/reference/common_attributes.md

```diff
@@ -0,0 +1,200 @@
+---
+$title: Common Attributes
+$order: 3
+---
+
+AMP provides a set of common attributes that are extended to many AMP components (and HTML elements).  This document describes each of the following attributes:
+
+* [fallback](#fallback)
+* [height](#height)
+* [heights](#heights)
+* [layout](#layout)
+* [media](#media)
+* [noloading](#noloading)
+* [on](#on)
+* [placeholder](#placeholder)
+* [sizes](#sizes)
+* [width](#width)
+
+## fallback
+
+The `fallback` attribute can be set on any HTML element, not just AMP elements. A fallback is a convention that allows the element to communicate to the reader that the browser does not support the element. If specified, a fallback element must be a direct child of the AMP element. The exact behavior with respect to the fallback is up to the element's implementation.
+
+Often used with: images, animations, audio, and videos
+
+Example:
+
+[sourcecode:html]
+<amp-anim src="animated.gif" width="466" height="355" layout="responsive" >
+  <div fallback>Cannot play animated images on this device.</div>
+</amp-anim>
+[/sourcecode]
+
+For more information, see [Placeholders & fallbacks](https://www.ampproject.org/docs/guides/responsive/placeholders).
+
+## height
+
+This attribute specifies the height of the element, in pixels. For some [layouts](https://www.ampproject.org/docs/guides/responsive/control_layout#the-layout-attribute), AMP components must have a `height` attribute that contains an integer pixel value.
+
+Example:
+
+[sourcecode:html]
+<amp-anim width="245"
+    height="300"
+    src="/img/cat.gif"
+    alt="cat animation">
+</amp-anim>
+[/sourcecode]
+
+For more information, see [Layout & Media queries](https://www.ampproject.org/docs/guides/responsive/control_layout) and the [Layout Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-layout.md).
+
+## heights
+
+All AMP elements that support the `responsive` layout, also support the `heights` attribute. The value of this attribute is a sizes expression based on media expressions, similar to the [img sizes attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) but with two key differences:
+
+
+ 1. The value applies to the height, not the width of the element.
+ 2. Percent values are allowed. A percent value indicates the percent of the element's width. For example, a value of `80%` indicates that the height of the element will be 80% of the element's width.
+
+When the `heights` attribute is specified along with `width` and `height`, the `layout` is defaulted to `responsive`.
+
+Example: 
+
+[sourcecode:html]
+<amp-img src="amp.png"
+    width="320" height="256"
+    heights="(min-width:500px) 200px, 80%">
+</amp-img>
+[/sourcecode]
+
+For more information, see [Art direction with srcset, sizes & heights](https://www.ampproject.org/docs/guides/responsive/art_direction).
```

Comment:
In this case, I'm going to avoid callouts as there's many FYIs will make the doc too busy.